### ISSUE TITLE
support for UV/DAD data?

### ISSUE THREAD

**Issue by ethanbass** [wkumler/RaMS#10]
Hi,

I was wondering if you would be interested in incorporating support for a function to extract DAD data from mzML files? I have some example files that were converted from the Thermo RAW format using the ThermoRawFileParser that encode both MS and DAD data. I figured out how to extract them using the MzR package, but I think I like you're approach better since it doesn't come with so many dependencies. If you want I could try to throw together a `grabMzmlUV` function and/or send over some example files.

Thanks!

**Comment by wkumler** [comment]
Hi Ethan,

I'd be definitely interested in expanding the package functionality to DAD data! I haven't worked with that type of MS at all so if you're able to send some and/or get started on a `grab` function that'd be welcome. Glad you like the package!

**Comment by ethanbass** [comment]
I took a first crack at it here: https://github.com/ethanbass/RaMs. 

And here is a link to the example files I have: https://cornell.box.com/v/thermo-mzml1

I think the solution i came up with is pretty specific to the way DAD data is encoded by Thermo, but I don't have any other example files, so I don't know if (or how) other manufacturers might differ.

I also changed an error to a warning in checkOutputQuality, because I didn't see why the whole function needs to error out if it can't find a certain data type -- like the DAD data for example. It seems like a warning is sufficient here, but feel free to revert if there's something I'm not seeing.
